.github/workflows/dusk-tests.yml

```
@@ -0,0 +1,100 @@
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+      
+      # Redis service
+      redis:
+        image: redis:alpine
+        ports:
+          - 6379:6379
+        options: --health-cmd="redis-cli ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+    - uses: actions/checkout@v3
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)